storage_service/templates/locations/pipeline_form.html

```diff
@@ -1,6 +1,7 @@
 {% extends "base.html" %}
+{% load i18n %}
 
-{% block page_title %}{{action}} Pipeline{% endblock %}
+{% block page_title %}{% blocktrans %}{{action}} Pipeline{% endblocktrans %}{% endblock %}
```

Comment:
I'm not sure about this either. I suspect the answer will be to pass the full string from the view to the template rather than attempt to construct it.